plugins/ocm-backend/src/helpers/config.ts

```diff
@@ -83,6 +84,7 @@ export const getHubClusterFromConfig = (
     serviceAccountToken: hub.getOptionalString('serviceAccountToken'),
     skipTLSVerify: hub.getOptionalBoolean('skipTLSVerify') || false,
     caData: hub.getOptionalString('caData'),
+    owner: hub.getOptionalString(OWNER_KEY) || 'unknown',
```

Comment:
If `hub` is resolved via `kubernetesPluginRef` by `getHubClusterFromKubernetesConfig()`, it has no owners field. This needs to be handed/passed down in `getHubClusterFromKubernetesConfig()`

Reply:
Once #228 is merged I'll push the change that will fix this together with updated tests.